src/ProjectOrigin.WalletSystem.IntegrationTests/GrpcTests.cs

```diff
@@ -0,0 +1,73 @@
+using Dapper;
+using FluentAssertions;
+using Grpc.Core;
+using ProjectOrigin.WalletSystem.IntegrationTests.TestClassFixtures;
+using ProjectOrigin.WalletSystem.Server;
+using ProjectOrigin.WalletSystem.Server.Database;
+using ProjectOrigin.WalletSystem.Server.Models;
+using ProjectOrigin.WalletSystem.V1;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ProjectOrigin.WalletSystem.IntegrationTests;
+
+public class GrpcTests : GrpcTestsBase
+{
+    private JwtGenerator _tokenGenerator;
+
+    public GrpcTests(GrpcTestFixture<Startup> grpcFixture, PostgresDatabaseFixture dbFixture, ITestOutputHelper outputHelper) : base(grpcFixture, dbFixture, outputHelper)
+    {
+        _tokenGenerator = new JwtGenerator();
+    }
+
+    [Fact]
+    public async Task CreateWalletSection_ValidResponse_InDatabase()
```

Comment:
We have not discussed naming conventions, but I find this naming convention difficult to use for integration tests. In certificates domain we have switched to using a descriptive name with words separated by underscores. So this method could be named, e.g. `can_create_wallet_section()`